test/e2e/networks/long.toml

```diff
@@ -0,0 +1,254 @@
+ipv6 = true
+initial_height = 2
+key_type = ""
+abci_protocol = "tcp"
+upgrade_version = "cometbft/e2e-node:latest"
+load_tx_size_bytes = 0
+load_tx_batch_size = 100
+load_tx_connections = 0
+
+[initial_state]
+  initial01 = "a"
+  initial02 = "b"
+  initial03 = "c"
+
+[validators]
+
+[validator_update]
+  [validator_update.0]
+    validator01 = 97
+    validator02 = 59
+    validator03 = 85
+    validator04 = 41
+    validator05 = 65
+  [validator_update.10]
+    validator06 = 66
+  [validator_update.15]
+    validator07 = 39
+  [validator_update.1005]
+    validator08 = 50
+  [validator_update.2005]
+    validator09 = 50
+  [validator_update.3005]
+    validator10 = 50
+
+[node]
+  [node.full01]
+    mode = "full"
+    version = "ghcr.io/informalsystems/tendermint-e2e-node:v0.34.x"
```

Comment:
How can we make this general for all branches?
Maybe the update should be from `cometbft/e2e-node:latest` to `cometbft/e2e-node:local`?

Reply:
Would this not actually break if merging into `main`? We'd be running a v0.34-based node alongside v0.38-based ones, right?

Reply:
Hi, sorry for delay coming back to this. I looked at @lasarojc's commits below (thanks for them!) and I think they address this concern.
I have also removed `backport-to-v0.34x` as that will need to be a special case.
Are we still missing something here?